setup.py

```diff
@@ -56,7 +60,7 @@ def run(self):
 
 # This dict is used for passing extra arguments that are setuptools
 # specific to setup
-setuptools_extra_args = {}
+setuptools_extra_args = {"python_requires": PYTHON_REQUIRES}
```

Comment:
For consistency, put this after the `if 'setuptools' in sys.modules:` guard below.

Reply:
Done

Reply:
Is there a reason for the two if statements, wouldn't be simpler if they were combined?

Reply:
No reason for the to ifs. It would be simpler to combine them.